tests/MSBuildDeviceIntegration/Tests/InstallAndRunTests.cs

```diff
@@ -1117,5 +1118,127 @@ public void MicrosoftIntune ([Values (false, true)] bool isRelease)
 				Path.Combine (Root, builder.ProjectDirectory, "logcat.log"), 30);
 			Assert.IsTrue (didLaunch, "Activity should have started.");
 		}
+
+		[Test]
+		public void GradleFBProj ([Values (false, true)] bool isRelease)
+		{
+			var moduleName = "Library";
+			var gradleTestProjectDir = Path.Combine (Root, "temp", "gradle", TestName);
+			var gradleModule = new AndroidGradleModule (Path.Combine (gradleTestProjectDir, moduleName));
+			gradleModule.PackageName = "com.microsoft.mauifacebook";
+			gradleModule.BuildGradleFileContent = $@"
+plugins {{
+    id(""com.android.library"")
+}}
+android {{
+    namespace = ""{gradleModule.PackageName}""
+    compileSdk = {XABuildConfig.AndroidDefaultTargetDotnetApiLevel}
+    defaultConfig {{
+        minSdk = {XABuildConfig.AndroidMinimumDotNetApiLevel}
+    }}
+}}
+dependencies {{
+    implementation(""androidx.appcompat:appcompat:1.6.1"")
```

Comment:
do the gradle and c# project need to have the same version? the C# side is 1.7.x

Reply:
Fair point I've updated this, best to keep these in sync